paper/src/main/java/com/denizenscript/denizen/paper/utilities/PaperAPIToolsImpl.java

```diff
@@ -122,6 +139,17 @@ public void sendSignUpdate(Player player, Location loc, String[] text) {
         player.sendSignChange(loc, components);
     }
 
+    @Override
+    public void sendSignUpdate(Player player, Location loc, String[] text, Side side) {

```

Comment:
Is there any reason this has a per-platfrom impl? the only platform specific code is setting the lines onto a `Sign`, which there's already `PaperAPITools#setSignLine` for - should be able to have this directly in the mechanism and just use that to set the lines